src/applications/appeals/995/pages/contactInformation.js

```diff
@@ -1,27 +1,27 @@
-import profileContactInfo from 'platform/forms-system/src/js/definitions/profileContactInfo';
-import set from 'platform/utilities/data/set';
+import profileContactInfo, {
+  profileReviewErrorOverride,
+} from 'platform/forms-system/src/js/definitions/profileContactInfo';
 import { getContent } from 'platform/forms-system/src/js/utilities/data/profile.js';
 
-import { contactInfoValidation } from '../../shared/validations/contactInfo';
+import { contactInfo995Validation } from '../../shared/validations/contactInfo';
+
+// Fix review & submit page chapter & page error highlighting when there's an
+// error; showing default settings
+export const profileReviewOverride = profileReviewErrorOverride({
+  contactInfoChapterKey: 'infoPages',
+  contactInfoPageKey: 'confirmContactInfo',
+  wrapperKey: 'veteran',
+});
 
 const allContacts = ['mobilePhone', 'homePhone', 'address', 'email'];
-const homelessContacts = ['mobilePhone', 'homePhone', 'email'];
 
 export default profileContactInfo({
   content: getContent('appeal'),
-  contactInfoRequiredKeys: [],
+  contactInfoRequiredKeys: allContacts,
```

Comment:
What's this do?

Reply:
If you look at the `updateSchema` callback that was removed (line 16), you'll see that it sets the page schema required fields based on the homeless answer. But our 995 form doesn't currently include a homeless question, so we're setting the `contactInfoRequiredKeys` to include `allContacts` instead of setting it via the callback.